### ISSUE TITLE
pb within basic PLOT on u22.04

### ISSUE THREAD

**Issue by alaingdl** [gnudatalanguage/gdl#1332]
I did switched my laptop info ubun22.04 ! yes :smile: 

Compiling GDL (git version) with the script + install, I do have problems on basic plots : 

```
./GDL/gdl-1.0.1git220525/install/bin/gdl -q
GDL> plot, findgen(10)
GDL> plot, histogram(randomu(seed, 10000))
% PLOTCoordinate system in error, please report to authors.
```
for the first plot, the window GDL 0 does appear, but just black (nothing plotted)
for the second one : xrange 0 -- 1 yrange 0 -- 1e4 (OK) but this message

```
./GDL/gdl-1.0.1git220525/install/bin/gdl -q --no-use-wx 
GDL> plot, findgen(10)
GDL> plot, histogram(randomu(seed, 10000))
% PLOTCoordinate system in error, please report to authors.
```
same problems

A.

**Comment by jtappin** [comment]
Hi Alain,
Looks like it can't cope with plotting an array of length 1.
````
GDL> help, histogram(randomu(seed, 10000))
<Expression>    LONG      = Array[1]
GDL> plot, [5]
% PLOTCoordinate system in error, please report to authors.
````

**Comment by alaingdl** [comment]
Hi James

You are obviously right but IDL 8.8 does allow that without message, and GDL also till a change between 211108 and 211116
(what is strange too it that IDL 8.7 gives a message at first call : 
% Program caused arithmetic error: Floating illegal operand )

See from me, maybe more useful to have a warning message in GDL saying "bad idea, array of length 1".

And my bug report is bad since two problems are reported at the same time

Alain

**Comment by GillesDuvert** [comment]
solved in #1338
